slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -269,6 +269,11 @@ class SLAMServiceImpl final : public SLAMService::Service {
     size_t current_file_offline = 0;
     std::string current_file_online;
 
+    // Enable Localization mode to utilize a cached version of the pointcloud
+    // map, to skip unecessary encoding steps
+    std::atomic<bool> localization_map_ready{false};
+    std::string localization_pointcloud_map;
```

Comment:
[q] Why not use `latest_pointcloud_map` instead of creating a new variable?

Reply:
I made a new variable because it's not the `latest_pointcloud_map`. I can rewrite some logic to use this instead